Document/0x05b-Basic-Security_Testing.md

```diff
@@ -66,6 +66,58 @@ Once you've configured the network and established a connection between the test
 
 After completing these steps and starting the app, the requests should show up in the interception proxy.
 
+As mentioned before starting at Android 7, android device won't trust user CA certificates. In the following we will be presenting two methods to bypass this android security mesure.
+
+#### Network Security Configuration
+
+Available since Android 7,this feature allows apps to customize their network security settings,by precising which CA certificates the app will be trusting.This CA certificates can be system trusted CAs as well as user non-public CAs or both.
```

Comment:
Can you change this in to: 
From Android 7 onward, the network security configuration allows apps to customize their network security settings, by defining which CA certificates the app will be trusting. The CA certificates trusted by the app can be a system trusted CA as well as a user non-public CA.